cfme/tests/middleware/test_provider_crud.py

```diff
@@ -0,0 +1,8 @@
+from cfme.middleware.hawkular import HawkularProvider
+from cfme import login
+
+login.login_admin()
+
+hawkular = HawkularProvider('My Hawkular', 'livingontheedge.hawkular.org', '80')
+
+hawkular.create(cancel=False, validate_credentials=False)
```

Comment:
All these things must be enclosed in test functions :)

```python
import pytest

@pytest.fixture(scope='function')
def hawkular_provider():
    return HawkularProvider(...)

def test_hawkular_crud(hawkular_provider):
    hawkular_provider.create(cancel=False, validate_credentials=False)
    # ...
```

Ideally, the data for the HawkularProvider should live in cfme_data.yaml and the test just call something like `HawkularProvider.from_config("config_name")`